README.md

```diff
@@ -1,21 +1,26 @@
 # eigen3_cmake_module
 
+On Ubuntu Bionic the `Eigen3` CMake package offers exported targets and non-standard CMake variables.
+This is a problem for packages using [ament_cmake](https://github.com/ament/ament_cmake).
```

Comment:
```suggestion
This is a problem for packages using [ament_cmake](https://github.com/ament/ament_cmake).

```
Just to break up the block. First paragraph subject is introducing the fact that there is a problem, second is the effects.